tokio-sync/src/lib.rs

```diff
@@ -27,10 +27,26 @@ macro_rules! if_fuzz {
     }}
 }
 
-pub mod lock;
+macro_rules! pin_mut {
```

Comment:
The current `futures_util` provides this: [futures_util::pin_mut](https://rust-lang-nursery.github.io/futures-api-docs/0.3.0-alpha.17/futures_util/macro.pin_mut.html)

Reply:
Good point, I couldn't tell if it was just futures-preview or futures-uril 😄

Reply:
sounds good to me!